Tasks/PythonScriptV0/pythonscript.ts

```diff
@@ -0,0 +1,71 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+
+import * as task from 'vsts-task-lib/task';
+import * as toolRunner from 'vsts-task-lib/toolrunner';
+
+import * as uuidV4 from 'uuid/v4';
+
+interface TaskParameters {
+    targetType: string,
+    filePath?: string,
+    script?: string,
+    arguments?: string,
+    pythonInterpreter?: string,
+    workingDirectory?: string,
+    failOnStderr?: boolean
+}
+
+export async function pythonScript(parameters: Readonly<TaskParameters>): Promise<void> {
+    // Get the script to run
+    const scriptPath = await (async () => {
+        if (parameters.targetType.toLowerCase() === 'filepath') { // Run script file
+            const filePath = parameters.filePath!; // Required if `targetType` is 'filepath'
+            if (!fs.statSync(filePath).isFile()) {
+                throw new Error(task.loc('NotAFile', filePath));
+            }
+            return filePath;
+        } else { // Run inline script
+            // Print one-line scripts
+            const script = parameters.script!; // Required if `targetType` is 'script'
+            if (!(script.includes('\n') || script.toLowerCase().includes('##vso['))) {
```

Comment:
I carried over the one-line restriction from the Bash task, but I don't see any reason for it -- the length of the script is set in the task.json anyway.  So I'm just going to log the whole inline script.

Reply:
Actually, the script gets logged when you run with system.debug, so I think I'll just remove this.